docs/griptape-framework/structures/tasks.md

```diff
@@ -146,3 +146,254 @@ agent.run()
                              Output: The summary of the webpage https://www.griptape.ai has been successfully stored in a file named
                              griptape.txt.
 ```
+
+
+## Extraction Task
+
+To extract information from a text, you can use the ExtractionTask. This task allows us to pass in an Extraction Engine and a set of arguments to the engine.
```

Comment:
Link to the reference docs for Extraction Engine.

Reply:
Done